javascript/organizing_your_javascript_code/classes.md

```diff
@@ -60,5 +60,8 @@ This section contains questions for you to check your understanding of this less
 * <a class="knowledge-check-link" href="https://javascript.info/property-accessors">What are "getters" & "setters"?</a>
 * <a class="knowledge-check-link" href="https://javascript.info/class">Describe computed names and class fields.</a>
 * <a class="knowledge-check-link" href="https://javascript.info/class">Describe function binding.</a>
+* <a class="knowledge-check-link" href="(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)">Describe static properties.</a>
+* <a class="knowledge-check-link" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes">Be able to use inheritance with classes.</a>
+* <a class="knowledge-check-link" href="https://rajaraodv.medium.com/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65">Briefly talk about the conflict in JS with functional programming and classes.</a>
```

Comment:
These two knowledge checks were removed in another PR https://github.com/TheOdinProject/curriculum/pull/24432/files. I don't think we need to add them back.